docs/specs/structured-planning-gate.md

```
@@ -0,0 +1,58 @@
+# Structured planning gate — no duplicate / shallow tasks
+
+Branch: `fix/structured-planning-no-dup-tasks` (worktree `.worktrees/structured-planning`, off `origin/main`).
+
+## Problem (user report)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)